### PR TITLE
QGIS Vector geometry - tables for alg parameters (part three)

### PR DESCRIPTION
**Pull request by havatv**
Goal: Use tables for algorithm parameters.
The QGIS Vector geometry algorithms from "Rectangles, Ovals, Diamonds (Fixed)" until (and including) "Simplify".

Ticket(s): #4429.   Following up #4629.

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -2929,94 +2969,175 @@ Buffer shape parameters are specified through attribute of the input layer.
...
-By default, Z values are not considered when detecting duplicate vertices.
-E.g. two vertices with the same X and Y coordinate but different Z values will still
-be considered duplicate and one will be removed. If the Use Z Value parameter is true,
-then the Z values are also tested and vertices with the same X and Y but different Z
-will be maintained.
+By default, Z values are not considered when detecting duplicate
+vertices.
+E.g. two vertices with the same X and Y coordinate but different Z
+values will still be considered duplicate and one will be removed.
+If the Use Z Value parameter is true, then the Z values are also
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
If the :guilabel;`Use Z Value` parameter is true, then the Z values are also
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 2]
```diff
@@ -2929,94 +2969,175 @@ Buffer shape parameters are specified through attribute of the input layer.
...
+     - ``USE_Z_VALUE``
+     - [boolean |dataDefined|]
 
-``Use Z value`` [boolean |dataDefined|]
-  Allows to consider the Z coordinate when detecting duplicate vertices ie two points
-  at the same X,Y coordinate but with different Z value are not set as duplicates.
+       Default: False
+     - Use also the Z coordinate when detecting duplicate vertices
+       (two points that share X and Y coordinates but have different
+       Z values are not considered duplicates).
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: harmonize this description with the one of the previous alg.

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 3]
```diff
@@ -3080,96 +3290,219 @@ Rotates feature geometries by the specified angle clockwise.
 The rotation occurs around each feature's centroid, or optionally
 around a unique preset point.
 
-|checkbox| Allows :ref:`features in-place modification <processing_inplace_edit>`
+|checkbox| Allows
+:ref:`features in-place modification <processing_inplace_edit>`
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: i'm not sure we need to split these. It's easier to read on a single line than on two. And we shouldn't be obsessed by the 80 characters limit; most of the screens can now display more, and a few characters beyond should be acceptable

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 4]
```diff
@@ -2929,94 +2969,175 @@ Buffer shape parameters are specified through attribute of the input layer.
...
-be considered duplicate and one will be removed. If the Use Z Value parameter is true,
-then the Z values are also tested and vertices with the same X and Y but different Z
-will be maintained.
+By default, Z values are not considered when detecting duplicate
+vertices.
+E.g. two vertices with the same X and Y coordinate but different Z
+values will still be considered duplicate and one will be removed.
+If the Use Z Value parameter is true, then the Z values are also
+tested and vertices with the same X and Y but different Z will be
+maintained.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Just copy and paste, so we must assume that this is correct.

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 5]
```diff
@@ -2929,94 +2969,175 @@ Buffer shape parameters are specified through attribute of the input layer.
...
+       duplicates
+   * - **Use Z value**
+     - ``USE_Z_VALUE``
+     - [boolean |dataDefined|]
 
-``Use Z value`` [boolean |dataDefined|]
-  Allows to consider the Z coordinate when detecting duplicate vertices ie two points
-  at the same X,Y coordinate but with different Z value are not set as duplicates.
+       Default: False
+     - Use also the Z coordinate when detecting duplicate vertices
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
     - If the :guilabel;`Use Z Value` parameter is true, then the Z values are also
       tested and vertices with the same X and Y but different Z will be
       maintained.
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 6]
```diff
@@ -2929,94 +2969,175 @@ Buffer shape parameters are specified through attribute of the input layer.
...
+       (two points that share X and Y coordinates but have different
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst (hunk 2 above)
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 7]
```diff
@@ -3221,30 +3591,76 @@ If no M values exist, the geometry will be upgraded to include M values.
...
+     - Type
+     - Description
+   * - **Input layer**
+     - ``INPUT``
+     - [vector: any]
+     - Input vector layer
+   * - **Raster layer**
+     - ``RASTER``
+     - [raster]
+
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 8]
```diff
@@ -3221,30 +3591,76 @@ If no M values exist, the geometry will be upgraded to include M values.
...
+       Default: []
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
```

**Comment by havatv** [comment]
Thanks for the review, @DelazJ!

**Pull request merged**